getambassadorio_blc.py

```diff
@@ -183,6 +189,7 @@ def _parse_srcset_value(attrvalue: str) -> List:
 
 def main(checkerCls: CheckerInterface, projdir: str, pages_to_check_file: str) -> int:
     urls = [
+        'http://localhost:9000/docs/telepresence/',
```

Comment:
this is only for testing, please remove it

Reply:
Done